src/systemcmds/param/param.cpp

```diff
@@ -714,7 +734,7 @@ do_show_print_for_airframe(void *arg, param_t param)
 }
 
 static int
-do_set(const char *name, const char *val, bool fail_on_not_found)
+do_set(const char *name, const char *val, bool fail_on_not_found, bool set_default)
```

Comment:
I don't love the API with two `bool`s and one of them defaulted. It's easy to get wrong.

What about?
```
enum class FailType : uint8_t {
    FailOnNotFound,
    DontFail
};
enum class SetType : uint8_t {
    Custom,
    Default,
};

int do_set(const char *name, const char *val, FailType fail_type, SetType set_type)
```

Reply:
This was mostly laziness. I'll add a completely separate helper that also gives more appropriate output. I also want to discourage regular users from ever touching it. It should be a special mechanism only used at boot by the system.

Reply:
Updated to a standalone do_set_custom_default with different output.